livekit-agents/livekit/agents/ipc/job_runner.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import annotations
+
+from typing import Any, Protocol
+
+from ..job import RunningJobInfo
+
+
+class JobRunner(Protocol):
```

Comment:
A little confusing reading the code that there is a `JobRunner` and a `JobExecutor`. Maybe this  becomes `JobExecutor` and the worker accepts a JobExecutor instance? For which we can have a default

Reply:
I'm using JobExecutorType